lib/membrane_opus/parser.ex

```diff
@@ -1,79 +1,254 @@
 defmodule Membrane.Opus.Parser do
   @moduledoc """
-  Parses self-delimiting Opus stream.
+  Parses a raw incoming Opus stream and adds caps information, as well as metadata.
 
-  The self-delimiting version of Opus packet contains all the information about frame sizes
-  needed to separate the packet from byte stream and decode it. This parser drops the last frame size
-  from each packet header, converting packets to the basic version. The remaining frame size needs
-  to be derived from entire packet size.
-  See https://tools.ietf.org/html/rfc6716#appendix-B for details.
+  Adds the following metadata:
+
+  frame_size :: 2.5 | 5 | 10 | 20 | 40 | 60
+    Number of milliseconds per frame encoded in this packet.
+
+  frame_lengths :: [non_neg_integer()]
```

Comment:
yeah - because ogg needs to know for a given packet how many MS it represents. to do this you need to multiple the number of frames w/in a packet by the frame-length. instead of coding it like this, I could just have an `int` of number of frames per packet, but this seemed like it could be useful and is given from the data. what do you think?

Reply:
I think it's better not to pass too much information if not necessary. I'd just pass what's required - the packet duration. BTW, you can also add the `:timestamp` field to metadata when not present - it's required by some elements, RTP serializer included, and it's going to be standardized (soon, I hope)

Reply:
what is `timestamp` supposed to hold? i'd definitely rather do that if that's gonna become the new standard, but can you descrbie a bit more?

Reply:
It holds the position of the packet in time, since the beginning of the stream, in `Membrane.Time` units

Reply:
FYI - simplifying this even further so no need for timestamp